_posts/2017-08-01-0.1-using-network-policy.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Using Network Policy in concert with Istio"
+overview: How Kubernetes Network Policy relates to Istio policy
+published: true
+permalink: blog/using-network-policy-in-concert-with-istio.html
+attribution: The Istio Team
+layout: post
+type: markdown
+---
+
+The use of Network Policy to secure applications running on Kubernetes is a now a widely accepted industry best practice.  Given that Istio also supports policy, we want to spend some time explaining how Istio policy and Kubernetes Network Policy interact and support each other to deliver your application securely.
+
+<!--end_excerpt-->
+
+Let’s start with the basics: why might you want to use both Istio and Kubernetes Network Policy? The short answer is that they are good at different things. Consider the main differences between Istio and Network Policy (we will describe “typical” implementations, e.g. Calico, but implementation details can vary with different network providers):
+
+|                       | Istio Policy | Network Policy     |
+| --------------------- | ------------ | ------------------ |
+| **Layer**             | "RPC" --- 7  | "Network" --- L3-4 | 
```

Comment:
Istio Policy can operate at the network layer as well, it is a superset of the network policy, so it gets all the advantages, with the added bonus for flexibility.